css/fonts.css

```diff
@@ -0,0 +1,76 @@
+/*--------------Header-------------------*/
```

Comment:
It's better to not declare where you font-families will be used in your stylesheet because what if this is not true in to future?